simulation/manage.py

```diff
@@ -0,0 +1,96 @@
+import csv
+import argparse
+import time
+
+from web3 import Web3, HTTPProvider
+from solc import compile_files
+
+
+def get_token_instance(compiled_source):
+    with open('deploy_info.csv', 'rt') as text_file:
+        spamreader = csv.reader(text_file, quoting=csv.QUOTE_MINIMAL)
+        next(spamreader)
+        next(spamreader)
+        next(spamreader)
+        token_address = next(spamreader)[1]
+
+    token_interface = compiled_source['../contracts/MFC_coin.sol:MFC_Token']
+    token_contract = w3.eth.contract(
+        abi=token_interface['abi'],
+        bytecode=token_interface['bin'])
+    token_instance = token_contract(token_address)
+    return token_instance
+
+
+def get_controller_instance(compiled_source):
+    with open('deploy_info.csv', 'rt') as text_file:
+        spamreader = csv.reader(text_file, quoting=csv.QUOTE_MINIMAL)
+        next(spamreader)
+        next(spamreader)
+        address = next(spamreader)[1]
+
+    ico_controller_interface = compiled_source['../contracts/ICO_controller.sol:ICO_controller']
+
+    ico_controller_contract = w3.eth.contract(
+        abi=ico_controller_interface['abi'],
+        bytecode=ico_controller_interface['bin'])
+    ico_controller_instance = ico_controller_contract(address)
+    return ico_controller_instance
+
+
+ap = argparse.ArgumentParser()
+
+ap.add_argument('--address', '-a', type=str, help='ICO controller address.', required=True)
+ap.add_argument('command', type=str, choices=['balance', 'whitelist'],
+                help='Command to do')
+
+
+def add_address_to_whitelist(address, controller_instance):
+    tx_hash = controller_instance.transact(
+        {'from': w3.eth.accounts[0]}
+    ).addBuyerToWhitelist(address)
+    while True:
+        try:
+            w3.eth.getTransactionReceipt(tx_hash)
+            break
+        except Exception as e:
+            print("Wait for account to be added to whitelist")
+            time.sleep(5)
+            continue
+
+    print("\n\n{} successfully added to whitelist".format(address))
+
+
+def print_address_balance(address, token_instance):
+    addr_balance = token_instance.call().balanceOf(address)
+    print("{} balance: {}".format(address, addr_balance))
+
+
+if __name__ == '__main__':
+    args = vars(ap.parse_args())
+    address = args['address']
+    command = args['command']
+
+    w3 = Web3(HTTPProvider('http://127.0.0.1:8545'))
+    w3.personal.unlockAccount(w3.eth.accounts[0], '1')
+
+    with open('deploy_info.csv', 'rt') as text_file:
+        spamreader = csv.reader(text_file, quoting=csv.QUOTE_MINIMAL)
+        next(spamreader)
+        next(spamreader)
+        next(spamreader)
+        token_address = next(spamreader)[1]
```

Comment:
It would be better, I think, use here function get_csv_file_row() that you wrote.